frontend/src/pages/performers/performerForm/PerformerForm.tsx

```diff
@@ -47,14 +49,14 @@ type OptionEnum = {
   disabled?: boolean;
 };
 
+const genderOptions = Object.keys(GenderEnum).map((g) => ({
+  value: g,
+  label: GenderTypes[g as GenderEnum],
+}));
+
 const GENDER: OptionEnum[] = [
   { value: "", label: "Select gender...", disabled: true },
-  { value: "null", label: "Unknown" },
```

Comment:
1. I don't think `GenderEnum` contains a mapping from `null` to "Unknown" and neither does `GenderTypes`.
2. My guess is the specific order of options was intentional here, unselected and unknown at the top, then the rest (maybe) in order of most-to-least prevalent genders. (my wording is a bit weird, what I mean is, Female is probably more common than Male, etc.)

I did not review the entire PR, just this as it caught my attention.

Reply:
Makes sense, will revert this change. Wanted to reduce the reliance on hardcoded gender enums, but from your description, it makes sense to have it hardcoded in this location.

Reply:
@peolic  just checked, the order does not change anything, you can take a look at StashDB, it's rendered in alphabetical order.
Will revert the delete of "null" because that one is necessary, but keep the spread syntax